scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl

```diff
@@ -231,7 +233,7 @@ task AssertIdenticalOutputs {
           actual="actual/$vcf"
           expected="expected/$vcf"
           set +o errexit
-          cmp <(grep '^#' $actual) <(grep '^#' $expected)
+          cmp <(grep '^#' $actual | grep -E -v '^##GATKCommandLine=') <(grep '^#' $expected | grep -E -v '^##GATKCommandLine=')
```

Comment:
Running `VariantFiltration` puts a `##GATKCommandLine` header in the headers which has different datetimes and filepaths for every run, so exclude this from consideration.